### ISSUE TITLE
NoMethodError produced

### ISSUE THREAD

**Issue by jacargentina** [Stonelinks/jekyll-gettext-plugin#1]
I have this error when trying to use this plugin:

```
/usr/local/share/gems/gems/jekyll-gettext-plugin-0.0.3/lib/jekyll/gettext/plugin.rb:50:in `block in process': undefined method `dest=' for #<Jekyll::Site:0x00000000a303e0> (NoMethodError)
    from /usr/local/share/gems/gems/jekyll-gettext-plugin-0.0.3/lib/jekyll/gettext/plugin.rb:47:in `each'
    from /usr/local/share/gems/gems/jekyll-gettext-plugin-0.0.3/lib/jekyll/gettext/plugin.rb:47:in `process'
    from /usr/local/share/gems/gems/jekyll-2.5.2/lib/jekyll/command.rb:28:in `process_site'
    from /usr/local/share/gems/gems/jekyll-2.5.2/lib/jekyll/commands/build.rb:56:in `build'
    from /usr/local/share/gems/gems/jekyll-2.5.2/lib/jekyll/commands/build.rb:34:in `process'
    from /usr/local/share/gems/gems/jekyll-2.5.2/lib/jekyll/commands/build.rb:18:in `block (2 levels) in init_with_program'
    from /usr/local/share/gems/gems/mercenary-0.3.5/lib/mercenary/command.rb:220:in `call'
    from /usr/local/share/gems/gems/mercenary-0.3.5/lib/mercenary/command.rb:220:in `block in execute'
    from /usr/local/share/gems/gems/mercenary-0.3.5/lib/mercenary/command.rb:220:in `each'
    from /usr/local/share/gems/gems/mercenary-0.3.5/lib/mercenary/command.rb:220:in `execute'
    from /usr/local/share/gems/gems/mercenary-0.3.5/lib/mercenary/program.rb:42:in `go'
    from /usr/local/share/gems/gems/mercenary-0.3.5/lib/mercenary.rb:19:in `program'
    from /usr/local/share/gems/gems/jekyll-2.5.2/bin/jekyll:20:in `<top (required)>'
    from /usr/local/bin/jekyll:23:in `load'
    from /usr/local/bin/jekyll:23:in `<main>'
```

**Comment by NickColley** [comment]
Related: https://github.com/pouchdb/pouchdb/issues/3112

I think we may be using a jekyll version that's incompatible with this library, any ideas? :)

**Comment by niccokunzmann** [comment]
Yes, agree. I have a solution that works for me:

```
module Jekyll
  class Site
    attr_accessor   :dest
  end
end
```

This can be put into the `_plugins` folder.

**Comment by niccokunzmann** [comment]
@nickcolley @jacargentina If you like, you can review #2.

**Comment by Stonelinks** [comment]
Just released @niccokunzmann's fix on 0.0.4: https://rubygems.org/gems/jekyll-gettext-plugin/versions/0.0.4
